index.html

```diff
@@ -269,11 +269,19 @@ <h3>Wacky Small Layouts</h3>
 
             <div class="section" id="sig_info">
                 <h2>Signature Info</h2>
+
+                <button type="button" name="preview" id="preview" disabled>Preview Output</button>
+                 <sub>(only tested in Chrome, Duplex only ATM)</sub><BR>
```

Comment:
FWIW it's working for me in Firefox. I'd change this text to 'Duplex-only right now, not tested in all browsers'

Reply:
Thanks for testing that!